src/services/SiteLaunchService.ts

```diff
@@ -0,0 +1,7 @@
+export const isUserUsingSiteLaunchFeature = (siteName: string): boolean => {
```

Comment:
hmm, i think a better way would be to just expose a `/features` endpoint on our backend and ask that the frontend queries this to see if we should render. 

something of this form could be like (on the fe): 
```ts
<FeatureFlag shouldRender = {...}>
  {our components}
</FeatureFlag>
```

and on the backend, we just expose a k/v mapping of features (`showSiteLaunch: true` or `siteLaunchRepos: [...]`)

this might be too heavy for this use case but 1 benefit is that we control the feature roll out (it's backend driven) whereas here, we have to rely on the user hard-refreshing + the browser refetching the data. 

Also pls do file a ticket for removal/followup for this feature flag!

Reply:
well agreed, we will be moving towards that direction with the introduction of launchdarkly, will be working on that in coming sprints, where we will fetch via sdk.

 i think in the time being, for this use case am fine with the hacky approach. wdyt?

Reply:
I actually didnt think of that, thanks for the suggestion @seaerchin! Since we will be moving towards that with the introduction of launchdarkly anyways, we can 'unhackify' this approach then!

Reply:
[Issue filed](https://isomeropengov.atlassian.net/browse/IS-238?atlOrigin=eyJpIjoiZWQ0YjVkNWY1YmJhNGNkYWI5MDBlOGY0YzVkN2ZhYjQiLCJwIjoiaiJ9)

Reply:
i think as long as 

1. we won't need to roll this back (ie, all future rollouts must be back compat with this) 
2. we have a clear deprecation plan (on the issue) 

this is ok